src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -164,12 +164,13 @@ public function get( WP_REST_Request $request ) {
 			/**
 			 * Filter the arguments used to get the events on the archive page via REST API.
 			 *
```

Comment:
Add a missing `@since` of when this param was added.

Reply:
This filter was added during this MR so `since TBD` is part of the dock block of this PR Let me move it a bit.